src/components/About/AboutUIModal.tsx

```diff
@@ -30,21 +30,20 @@ class AboutUIModal extends React.Component<AboutUIModalProps, AboutUIModalState>
   };
 
   render() {
+    const uiVersion =
+      process.env.REACT_APP_GIT_HASH === ''
+        ? process.env.REACT_APP_VERSION
+        : `${process.env.REACT_APP_VERSION} (${process.env.REACT_APP_GIT_HASH})`;
+    const coreVersion =
+      this.props.status[KIALI_CORE_COMMIT_HASH] === ''
```

Comment:
A check for `this.props.status ?` was being made, not sure when is unavailable and it shows 'Unknown' instead.
Can you add that back?

Reply:
I don't think it is needed, from what TS is giving me, this.props.status is `{ [key]: prop }`, not `{ [key]: prop } | undefined`.